### ISSUE TITLE
[Accordion] Unable to focus with Keyboard in Safari and Firefox

### ISSUE THREAD

**Issue by rafibomb** [zurb/foundation-sites#8549]
**How can we reproduce this bug?**

Use keyboard to Tab to an accordion. 

**What did you expect to happen?**

Accordion should get focus.

**What happened instead?**

Accordion is completely skipped.

**Test case:**

See test: accordion -> focus.html

**Comment by Owlbertz** [comment]
It's working for me with Firefox 44 on Ubuntu 14.04. Unfortunately I cannot test this in Safari atm. What's your Firefox version that creates the issue? @rafibomb

**Comment by sethkane** [comment]
I am going to check this out later this week.  I will keep you all posted.

**Comment by sethkane** [comment]
@rafibomb I just tested this in Chrome, Safari and Firefox and here is what I found:

Safari there are no issue but you need to understand that safari default doesn't allow for navigating links via the "Tab" key by default.  If you are on a mac try using option + tab and you can navigate all focusable objects like I did in this video: https://www.dropbox.com/s/tqkhgid7fadwqdf/zurb-accordion.mov?dl=0

Firefox on a mac for some reason disabled the "tab" key as a way to navigate links as well.  As far as I know there is no way around this without putting tabindex on every element which isn't ideal. I went to about 10 different website in firefox and I can't "tab" to anything other than form fields and <button> tags.  No a-href is tabable. Check out my video of Southwest.com website I can only tab to a small amount of items and no links: https://www.dropbox.com/s/lq8i2ro8ns0z9gx/southwest.mov?dl=0

My understanding is there is no bug but instead a defect in the way the browser works.

**Comment by kball** [comment]
@rafibomb what do you think?  Can we close this issue?

**Comment by sethkane** [comment]
I think so ;-)

**Comment by joe-watkins** [comment]
@rafibomb I've confirmed things work as expected in Safari and Firefox in this [standalone demo](http://s.codepen.io/joe-watkins/debug/PzWQyW#panel1d).

Safari:
http://www.screencast.com/t/NQSlT2NqoOw

Firefox:
http://www.screencast.com/t/FCHnG3DyuC

**Comment by kball** [comment]
Closing this based on the above discussion.
